handlebars-humanize/src/main/java/com/github/jknack/handlebars/HumanizeHelper.java

```diff
@@ -27,6 +27,8 @@
 
 import org.apache.commons.lang3.LocaleUtils;
```

Comment:
true, was also present in the other class - usually my IDE removes unused imports automatically, but for this project this wasn't configured. sorry.